sentences/de/climate_HassClimateSetTemperature.yaml

```diff
@@ -25,9 +25,9 @@ intents:
           - "(<area_heizung>|<floor_heizung>) auf <temperature>[ <setzen_end_of_sentence>]"
           - "<area_floor> <temperature>"
           - "heiz[e] <area> auf <temperature>[ auf]"
-          - "heiz[e] [<artikel_bestimmt> ]<floor> auf <temperature>[ auf]"
+          - "heiz[e] <floor> auf <temperature>[ auf]"
           - "kühl[e] <area> auf <temperature>[ (ab|[he]runter)]"
-          - "kühl[e] [<artikel_bestimmt> ]<floor> auf <temperature>[ (ab|[he]runter)]"
+          - "kühl[e] <floor> auf <temperature>[ (ab|[he]runter)]"
```

Comment:
Why remove the article here? It is more common to say `heize das obergeschoss auf 12 ° auf" then ommitting `das`.